src/brackets/index.js

```diff
@@ -4,6 +4,60 @@
  * @param {string} str The string of brackets.
  * @returns {"valid" | "invalid"} Whether or not the string is valid.
  */
-function isValid(str) {}
+function isValid(str) {
+let char=["(", ")", "[", "]", "{","}"]
+let a=[...str]
+let b1=["(","[","{"]
+let b2=[")","]","}"]
+let c1=[]
+let c2=[]
+let u=[]
+for(var i=0;i<a.length;i++){
+if(b1.indexOf(a[i])>-1){
+  c1.push(a[i])
+}
+if(b2.indexOf(a[i])>-1){
+  c2.push(a[i])
+}
+if(char.indexOf(a[i])%2==0){
+    let p1=i
+    let p2=a[i+2]
+    let p3=a[i+1]
+    if(p2==char[char.indexOf(a[i])+1]&&p3!=char[char.indexOf(a[i])+1]&&p3!==a[i]){
+      u.push("yes")
+    }
+  }
+}
+var y1=[]
```

Comment:
Stick to a using `let` or `const` for variable declaration.

Reply:
Thanks @bondz   i will strictly adhere to that in the subsequent labs.